docs/example/manifests/access/traffic-access-v1.yaml

```diff
@@ -19,8 +19,8 @@ spec:
     name: bookbuyer
     namespace: bookbuyer
   #- kind: ServiceAccount
-    #name: bookthief
-    #namespace: bookthief
+     #name: bookthief
```

Comment:
I tested this change with command+/, which resolves to an extra space. Reverting this update works as expected.